Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM  mhart/alpine-node:11.0.0
+
+# install dependencies
+RUN apk add git
+
+# build gitmoji-changelog from source
+COPY . /usr/src/gitmoji-changelog
+WORKDIR /usr/src/gitmoji-changelog
+RUN yarn
+
+# this folder is the target of the host folder that contains git repository
+WORKDIR /usr/src/app
```

Comment:
This instruction is not necessary as nothing is written is this folder. There is no guideline which stats where source folder should be created when working with sdk container, so nothing should be provisionned. The use could create an `/app` folder and there is nothing against it.

Reply:
Oh nice !
I didn't know :smile: 

I will update the Dockerfile ASAP.

Thank's for review.